packages/samples/mvp-dapp/src/views/tx.view.ts

```diff
@@ -0,0 +1,26 @@
+import { CellChangeData } from '@ckb-js/kuai-models'
+import { Cell, helpers, commons } from '@ckb-lumos/lumos'
+
+export class Tx {
+  static async toJson(
+    txSkeleton: helpers.TransactionSkeletonType,
```

Comment:
Is the `skeleton` available for unlocking the omnilock?

Reply:
Any update on this question?

Reply:
Supported, in https://github.com/ckb-js/kuai/pull/127/commits/cdb1aad571e0b66fe51ebd850896d4b5016b3a97.